lib/src/api.dart

```diff
@@ -259,6 +259,21 @@ class API extends Api {
     return resp['data'];
   }
 
+    Future<Map<String, dynamic>> getPaginatedActionsByWalletAddress(
+    String walletAddress,
+    int pageIndex, {
+    int updatedAt = 0,
```

Comment:
@artman-industries Can you remove this argument?

Reply:
done